text/0073-usage-of-transaction-types.md

```diff
@@ -0,0 +1,229 @@
+- `auto.file-io`
+- `auto.app-start`
+- `auto.jetpack-compose`
+
+```json
+"trace": {
+    "trace_id": "40072a6227d648449aa8665307a1fde3",
+    "span_id": "f2e763bf95c640df",
+    "op": "ui.load",
+    "status": "ok",
+    "exclusive_time": 23.461104,
+    "hash": "e2839639c27b6393",
+    "sampled": "true",
+    "start_timestamp": 1679374744.0518713,
+    "timestamp": 1679374744.6143088,
+    "type": "trace",
+    "origin": "auto.ui-view-controller",
+}
+```
+
+```json
+{
+  "spans": [
+    {
+      "start_timestamp": 1588601261.481961,
+      "description": "loadView",
+      "timestamp": 1588601261.488901,
+      "parent_span_id": "f2e763bf95c640df",
+      "trace_id": "40072a6227d648449aa8665307a1fde3",
+      "op": "ui.load",
+      "span_id": "b01b9f6349558cd1",
+      "origin": "auto.ui-view-controller",
+    },
+    {
+      "start_timestamp": 1588601261.535386,
+      "description": "BYZ-38-t0r-view-8bC-Xf-vdC.nib",
+      "timestamp": 1588601261.544196,
+      "parent_span_id": "f2e763bf95c640df",
+      "trace_id": "40072a6227d648449aa8665307a1fde3",
+      "op": "file.read",
+      "span_id": "b980d4dec78d7344",
+      "origin": "auto.file-io",
+    },
+    {
+      "timestamp": 1679374744.587838,
+      "start_timestamp": 1679374744.587426,
+      "exclusive_time": 0.411987,
+      "description": "calculatePI",
+      "op": "ui.load",
+      "span_id": "800b9c31b7f34ba2",
+      "parent_span_id": "f2e763bf95c640df",
+      "trace_id": "40072a6227d648449aa8665307a1fde3",
+      "status": "ok",
+      "origin": "manual",
+    },
+  ]
+}
+```
+
+### Pros <a name="option-5-pros"></a>
+
+1. Helps users to understand which parts of transactions where auto or manually instrumented.
+2. This addition can help the performance product to build new features and performance issues.
+
+### Cons <a name="option-5-cons"></a>
+
+1. Most of the time, the spans already contain enough information to know if they were auto or manually created. The extra property is redundant in most cases.
+2. Doesn't give insight into which types of transactions/spans our users are interacting with.
+3. Extends protocol and data structures.
+
+## Option 6: Trace Context <a name="option-6"></a>
+
+Same as [option 5](#option-5), but add the `origin` property to the [trace context](https://develop.sentry.dev/sdk/event-payloads/contexts/#trace-context).
+
+```json
+{
+  "contexts": {
+    "trace": {
+      "op": "navigation",
+      "description": "User clicked on <Link />",
+      "trace_id": "743ad8bbfdd84e99bc38b4729e2864de",
+      "span_id": "a0cfbde2bdff3adc",
+      "status": "ok",
+      "origin": "auto.user-interaction-tracing",
+    }
+  }
+}
+```
+
+### Pros <a name="option-6-pros"></a>
+
+1. Helps users to understand which transactions where auto or manually instrumented.
+2. This addition can help the performance product to build new features and performance issues.
+
+### Cons <a name="option-6-cons"></a>
+
+1. Only works for transactions.
```

Comment:
Nope, my suggestion was that Option 6 is actually Option 5 + 6, so we are able to track the transaction and span `origin`.

Reply:
Sorry about the misunderstanding. I updated the RFC.